src/App.ts

```diff
@@ -1,47 +1,52 @@
+import { AppConfig } from './common/AppConfig';
 import { Application } from 'pixi.js';
+import { Settings } from './types/App.types';
+import { SplashPage } from './ui/pages/SplashPage';
+import { Events } from './common/Events';
 
-interface Settings {
-    background: {
-        color: string,
-    },
-    dimensions: {
-        width: number,
-        height: number,
-    },
-}
 
+/**
+ * The entry point.
+ * 
+ * The constructor initialises the AppConfig -to store the app setting globally, then the app.
+ */
 class App {
     private app: Application;
-    private settings: Settings = {
-        background: {
-            color: '#333333',
-        },
-        dimensions: {
-            width: 1024,
-            height: 600,
-        },
-    };
```

Comment:
migrated to `App.types.ts`
https://github.com/irgat/soloTest/blob/850758dbc02694d5a3ffd3992384460258d5a57a/src/types/App.types.ts#L13-L23